### PR TITLE
entry P

### PR DESCRIPTION
**Pull request by planteater**
asdfsdfg

[Changes reviewed on CodeStream](https://pd-api.codestream.us/r/Xz13RtJpEgJ-jDbv/RDJrU8cuSbiQPNsb5-gZyg?src=GitHub) by dave275 on Aug 27, 2020

### CODE REVIEW

File: freezer.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { SelectionDecorationProvider } from "./providers/selectionDecorationProv
 import { SetServerUrlRequestType} from "./protocols/agent/agent.protocol";
 // import { WebviewSidebarActivator } from "./views/webviewSidebarActivator";
 
+// entry point test
```
Comment (codestreamtest) [inline freezer.js:21]: single mingle

Comment (codestreamtest) [inline freezer.js:21]: start a rev pls

File: freezer.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ import { SelectionDecorationProvider } from "./providers/selectionDecorationProv
...
 export class Container {
 	static async initialization(
 		context: ExtensionContext,
```
Comment (codestreamtest) [inline freezer.js:24]: add to rev pls

Comment (codestreamtest) [reply freezer.js:24]: rev repl

**Comment by planteater** [comment]
Comment from GH.

File: freezer.js (hunk 2 above)
Comment (planteater) [reply freezer.js:24]: reply from GH web

**Pull request closed without merging**